vscode-wpilib/resources/webviews/projectcreator.html

```diff
@@ -2,61 +2,182 @@
-  <br/>
-  <input id="projectFolder" type="text" class="error" style="width: 100%; margin-top: 10px; margin-bottom: 10px" />
-  <br/>
-  <button id="projectSelectButton" data-i18n-trans>Select a new project folder</button>
-  <br/>
-  <br/>
-  <div id="projectnamediv" data-i18n-trans="ui" class="error" STYLE="font-weight:bold">Project Name</div>
-  <input id="projectName" type="text" class="error" style="width: 100%; margin-top: 10px; margin-bottom: 10px"
-  />
-  <br/>
-  <br/> <span data-i18n-trans STYLE="font-weight:bold">Create a new folder?</span>
-  <input id="newFolderCB" type="checkbox" checked/>
-  <br/>
-  <span data-i18n-trans>This creates a new folder at <i>Base Folder\Project Name</i>. Highly recommended to be checked. Otherwise the folder will be placed at <i>Base Folder</i> and not utilize the Project Name</span>
-  <br/>
-  <br/>
-  <div id="teamnumberdiv" class="error" data-i18n-trans="ui" STYLE="font-weight:bold">Team Number</div>
-  <input id="teamNumber" class="error" type="text" style="width: 100%; margin-top: 10px; margin-bottom: 10px"
-  />
-  <br/>
-  <div id="desktopdiv">
-    <span data-i18n-trans STYLE="font-weight:bold">Enable Desktop Support</span>
-    <input id="desktopCB" type="checkbox" />
-    <br/>
-    <span data-i18n-trans data-i18n-key="hintDesktopSupport">
-      This is needed for simulation and unit testing support, however there are some cases where this will do
-      some unexpected things during build. In addition, not all vendor libraries support desktop.
-      This option can be set with the command "WPILib: Set Desktop Support" at any time.
-    </span>
-  </div>
-  <br/>
+  <div class="project-container">
+    <img src="replaceresource/resources/wpilib-generic.svg" height="75" />
+
+    <h1 class="project-title">WPILib Project Creator</h1>
+
+    <div class="wizard-progress">
+      <div class="progress-step active" data-step="1">Project Type</div>
+      <div class="progress-step" data-step="2">Project Settings</div>
+      <div class="progress-step" data-step="3">Location & Config</div>
+      <div class="progress-step" data-step="4">Review & Create</div>
+    </div>
+
+    <!-- Step 1: Project Type -->
+    <div class="wizard-step active" id="step-1">
+      <div class="step-header">
+        <h2>Step 1: Select Project Type</h2>
+        <p>Choose whether to create a project from a template or an example.</p>
+      </div>
+
+      <div class="selection-cards">
+        <div class="selection-card" data-value="Template">
+          <h3>Template</h3>
+          <p>Start with a basic robot program structure</p>
+          <div class="card-icon">
+            <svg viewBox="0 0 24 24" width="48" height="48">
+              <path fill="currentColor"
+                d="M14,17H7V15H14M17,13H7V11H17M17,9H7V7H17M19,3H5C3.89,3 3,3.89 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V5C21,3.89 20.1,3 19,3Z" />
+            </svg>
+          </div>

```

Comment:
Looks like we're losing localization in this file. Pretty sure we want it, only thing I'm not sure is if it's actually translated at all.